### DISCUSSION TITLE
Audio Unit v3 support for iOS

### DISCUSSION THREAD

**Discussion by pierreguillot** [pierreguillot/Camomile#278] [Ideas]

**Comment by rafagan** [comment]
This was implemented?

**Comment by pierreguillot** [comment]
No, the issue is still open.

**Comment by rafagan** [comment]
Do you know if is possible to use Pure Data inside AUAudioUnit?

**Comment by pierreguillot** [comment]
You can do that with Camomile

**Comment by rafagan** [comment]
Wow, so I can get a .pd file and run it inside garage band per example using camomile?

**Comment by pierreguillot** [comment]
If it's not related to AUv3 and iOS, and if you want help or discuss things, please use the discussions https://github.com/pierreguillot/Camomile/discussions to avoid too much noise in the issue section.

**Comment by rafagan** [comment]
That is straight related to AUV3 and iOS. There's some code example of how can I use Camomile inside AUV3 and play it on and host app?

**Comment by pierreguillot** [comment]
Camomile only supports AUv2 for the moment. Please create a discussion.
